src/test/java/org/minions/devfund/chelo/SquarelotronTest.java

```diff
@@ -0,0 +1,232 @@
+package org.minios.devfund.chelo.Squarelotron;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "SquarelotronTest.java" should be located in "org/minios/devfund/chelo/Squarelotron" directory, not in "/home/travis/build/Minions-F/devfund/src/test/java/org/minions/devfund/chelo". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)